pkg/scheduler/plugins/nodeorder/nodeorder.go

```diff
@@ -315,6 +323,12 @@ func (pp *nodeOrderPlugin) OnSessionOpen(ssn *framework.Session) {
 	p, _ = tainttoleration.New(nil, handle)
 	taintToleration := p.(*tainttoleration.TaintToleration)
 
+	ptsArgs := &config.PodTopologySpreadArgs{
+		DefaultingType: config.SystemDefaulting,
```

Comment:
make clear whether the config.SystemDefaulting impact batch workload without configuring topologySpread.

Reply:
This default policy has action `ScheduleAnyway` when unsatisfiable, so it will not affect filter process, and the default  `maxSkew` value has a little affect on score process.